tests/exporters/exporters_utils.py

```diff
@@ -110,6 +110,7 @@
     "speech-to-text": "hf-internal-testing/tiny-random-Speech2TextModel",
     "xlm": "hf-internal-testing/tiny-random-XLMModel",
     "xlm-roberta": "hf-internal-testing/tiny-xlm-roberta",
+    "vision-encoder-decoder": "hf-internal-testing/tiny-random-VisionEncoderDecoderModel-vit-gpt2",
```

Comment:
Could you add a test for `trocr` and `donut-swin` as well?

Reply:
added for trocr, donut-skin has threshold issue while exporting would look into it separately

Reply:
Then could we not add it in tasks.py? And handle adding donut-swin in a separate PR altogether?

Reply:
The test model has high atol requirement (4e-2). The export for the model in doc is slow but works with 1e-3 threshold, so not putting that here.

Reply:
What do you mean by `the model in doc`?

Reply:
I'd just like that what is said to be supported in `tasks.py` is actually supported, and we can make sure of it by adding an unit test.

Reply:
Commented Donut. Will handle this separately